dev-libs/xqilla/xqilla-2.3.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+MY_P="XQilla-${PV}"
+
+DESCRIPTION="An XQuery and XPath 2 library and command line utility written in C++"
+HOMEPAGE="http://xqilla.sourceforge.net/HomePage"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.gz"
+LICENSE="Apache-2.0 BSD"
+SLOT="0/2"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples faxpp static-libs tidy"
+
+# XQilla bundles two libraries:
+# - mapm, heavily patched
+# - yajl, moderately patched
+# There's currently no way to unbundle those
+
+RDEPEND=">=dev-libs/xerces-c-3.2.2
+	faxpp? ( dev-libs/faxpp )
+	tidy? ( app-text/tidy-html5 )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )"
+
+PATCHES=(
+	"${FILESDIR}/2.2.4-respect-ldflags-no-rpath.patch"
+	"${FILESDIR}/${P}-tidy.patch"
+)
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	default
+	mv configure.{in,ac} || die
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--with-xerces="${EPREFIX}"/usr \
+		$(use_enable debug) \
+		$(use_with tidy tidy /usr) \
+		$(use_with faxpp faxpp /usr) \
+		$(use_enable static-libs static)
+}
+
+src_compile() {
+	default
+	use doc && emake docs devdocs
+}
+
+src_install () {
+	use doc && HTML_DOCS=( docs/{dev-api,dom3-api,simple-api} )
+	default
+
+	if ! use static-libs; then
```

Comment:
Do you really need `.la` files for static libs?

Reply:
Probably not, I just tried to do this with minimal changes to the existing ebuild.

Reply:
> Please also include taking over maintenance and removing the p.mask (in separate commits).

I don't think I want to take over the maintenance.